apps/browser/qml/pages/components/BrowserListItem.qml

```diff
@@ -47,6 +47,8 @@ ListItem {
 
             width: parent.width - icon.width
             anchors.verticalCenter: parent.verticalCenter
+            leftMargin: Theme.paddingMedium
+            rightMargin: 0
```

Comment:
does it make any sense?

Reply:
It's a good question. I noticed that the `ValueButton` is inside a `Row` with the following properties:
```
        width: parent.width - 2*Theme.horizontalPageMargin
        x: Theme.horizontalPageMargin
```
So there is already a right margin of `Theme.horizontalPageMargin` applied to the item. If I'm understanding this code correctly, setting `rightMargin: 0` should leave it with a right margin of `Theme.horizontalPageMargin`. Please correct me if this is wrong or not what's needed.

Reply:
This is correct analysis as far as I see. ValueButton itself has a Column which is anchorized rightMargin bound to parent right.